src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1105,10 +1107,12 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         $message = ''
                                         $prefix = ''
 
-                                        # Don't show line information if script module
-                                        if (($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $err.CategoryInfo.Category -eq 'ParserError') -and !($myinv.ScriptName.EndsWith('.psm1', [System.StringComparison]::OrdinalIgnoreCase))) {
+                                        # The checks here determine if we show line detailed error information:
+                                        # - check if `ParserError` and comes from PowerShell which eventually results in a ParseException, but during this execution it's an ErrorRecord
```

Comment:
It's not a "ParseException" probably because of the following code before calling `Get-ConciseViewPositionMessage`.
A `ParserException` will fall in this if block, which will set `$err` to be the `ParserException.ErrorRecord` member.

https://github.com/PowerShell/PowerShell/blob/476af582d56b25913d1ff9dc8057e1a484f814a9/src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs#L1278-L1281

You may want to re-evaluate if you need to do something about the remote error in the above if block instead.

Reply:
This is why during the investigation I passed the original `$_` to the function instead of the reassigned `$err` and even stepping in the debugger, it wasn't a `ParseException`.